scripts/slate.sh

```diff
@@ -0,0 +1,79 @@
+#!/usr/bin/env bash
+set -euo pipefail
+
+if [ "$CIRCLE_BRANCH" == "" ]; then
+	echo "this script is meant to be run on CircleCI, exiting"
+	echo 1
+fi
+
+yes yes | git fetch origin develop
+
+# check for changes in the `rpc/core` directory
+did_rpc_change=$(git diff --name-status $CIRCLE_BRANCH develop | grep rpc/core)
+
+if [ "$did_rpc_change" == "" ]: then
+	echo "no changes detected in rpc/core, exiting"
+	exit 0
+else
+	echo "changes detected in rpc/core, continuing"
+fi
+
+# only run this script on changes to rpc/core committed to develop
+if [ "$CIRCLE_BRANCH" != "develop" ]; then
+	echo "the branch being built isn't develop, exiting"
+	exit 0
+else
+	echo "on develop, building the RPC docs"
+fi
+
+# godoc2md used to convert the go documentation from
+# `rpc/core` into a markdown file consumed by Slate
+go get github.com/melekes/godoc2md
```

Comment:
I think we can use the upstream one again: https://github.com/davecheney/godoc2md

anton's changes got merged in.